de.metas.business/src/main/java/de/metas/bpartner/impexp/BPartnerImportTableSqlUpdater.java

```diff
@@ -64,10 +66,19 @@ public void updateBPartnerImtortTable(@NonNull final String whereClause)
 
 		dbUpdateAdUserIdsFromExisting(whereClause);
 
-		dbUpdateCbPartnerIds(whereClause);
+		dbUpdateCbPartnerIdsFromValue(whereClause);
```

Comment:
Imho `ExternalId` (if set) needs to have higher prio than `Value`,
so 
`dbUpdateCbPartnerIdsFromValue(whereClause);`
and 
`dbUpdateCbPartnerIdsFromC_BPartner_ExternalId(whereClause);`

need to be the other way aground.
Agree?

Reply:
:+1: